### DISCUSSION TITLE
Chart Gallery Tag Updates

### DISCUSSION THREAD

**Discussion by yousoph** [apache/superset#27557] [Ideas]
Proposing a few updates to the chart/viz gallery to improve clarity and ease of use: 
![image](https://github.com/apache/superset/assets/10627051/8e465300-9545-47ba-a637-f80c0e7c4955)

1. Tag Name updates
* Update "Popular" -> "Featured" since Popular gives the impression that the section updates periodically based on usage 

2. Rename/Remove Recommended tags
* Remove or rename ECharts: open to suggestions but I'm not sure ECharts as a category is a useful / descriptive grouping to be a recommended tag for users 
* Remove Advanced-Analytics from the Recommended tags section or rename to be more descriptive: "Predictive" or "Comparison" are more easily understood than Advanced-Analytics, especially for newer users 

3. Reduce the number of tags on the left hand side by removing the following tags: 
* Aesthetic
* Coordinates
* Deprecated 
* Description
* Discrete
* Experimental
* Vertical 

4. Remove some specific tags from charts: 
* From Bar Chart (legacy), remove the "Popular" tag 

5. Collapse the "Tags" section on the left hand sidebar by default, to allow users to better focus on the Recommended tags and Category tags sections

**Comment by michael-s-molina** [comment]
These all seem reasonable to me.

One idea that could help with improving simplicity and highlighting the featured charts:
- Remove the Recommended section
- Move Featured as a sibling of All charts and make it selected by default
- Move ECharts or Predictive/Advanced Analytics to Tags if you think they matter
- Reduce the number of tags and categories to more meaningful ones

**Comment by yousoph** [comment]
https://github.com/apache/superset/pull/27708
